experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/Middleware/HandoffMiddleware.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.Bot.Builder;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using System.Linq;
+using LivePersonAgentBot.Bots;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Text;
+using System.Collections.Concurrent;
+using Newtonsoft.Json.Linq;
+
+namespace LivePersonAgentBot
+{
+    public class HandoffMiddleware : Microsoft.Bot.Builder.IMiddleware
```

Comment:
The very first using is "Microsoft.Bot.Builder", so I'm not clear on why the full namespace is here on IMiddleware. 

Consider running the standard .Net linting / rosyln analyzers over this code.